src/Compilers/Core/Desktop/AnalyzerFileReference.InMemoryAssemblyLoader.cs

```diff
@@ -150,14 +150,20 @@ private static Assembly LoadCore(string fullPath)
             /// </summary>
             private static Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEventArgs args)
             {
-                if (args.RequestingAssembly == null)
-                {
-                    return ResolveForUnknownRequestor(args.Name);
-                }
-                else
+                try
                 {
-                    return ResolveForKnownRequestor(args.Name, args.RequestingAssembly);
+                    if (args.RequestingAssembly == null)
+                    {
+                        return ResolveForUnknownRequestor(args.Name);
+                    }
+                    else
+                    {
+                        return ResolveForKnownRequestor(args.Name, args.RequestingAssembly);
+                    }
                 }
+                catch { }
```

Comment:
:exclamation: Since empty catch-all blocks are generally considered to be a bad practice, can you add a comment within the `{ }` explaining why an exception needed to be made here?